packages/plugins/rubygem-smart_proxy_discovery/rubygem-smart_proxy_discovery.spec

```
@@ -109,6 +81,9 @@ mv %{buildroot}%{gem_instdir}/settings.d/discovery.yml.example \
 %doc %{gem_instdir}/README.md
 
 %changelog
+* Thu Mar 07 2024 nofaralfasi <nalfassi@redhat.com> 1.0.5-10
+- Update to 1.0.5-10
```

```suggestion
- Regenerate spec file based on the latest template
```

(or something along these lines)

same for the commit message, then we're good to go :)